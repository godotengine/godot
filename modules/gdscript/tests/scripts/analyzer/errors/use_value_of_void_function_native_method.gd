func test():
	var obj := Node.new()
	print(obj.free()) # Native type method.
