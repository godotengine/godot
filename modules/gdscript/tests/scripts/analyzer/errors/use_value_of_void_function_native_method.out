GDTEST_ANALYZER_ERROR
>> ERROR at line 3: Cannot get return value of call to "free()" because it returns "void".
